src/python/binary_configurator.py

```diff
@@ -331,9 +327,6 @@ def main():
     # TX Params
     parser.add_argument('--tlm-report', type=int, const=240, nargs='?', action='store', help='The interval (in milliseconds) between telemetry packets')
     parser.add_argument('--fan-min-runtime', type=int, const=30, nargs='?', action='store', help='The minimum amount of time the fan should run for (in seconds) if it turns on')
-    parser.add_argument('--uart-inverted', dest='uart_inverted', action='store_true', help='For most OpenTX based radios, this is the default')
```

Comment:
We should keep these flags supported, otherwise configurator will not be able to use this script without updates. It will throw an error if unrecognized cli flag is passed.

Reply:
Good call. I've added an action so we can deprecate them.